src/react/components/Message.jsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React, {
+  useRef,
+  // useState,
+  forwardRef,
+  useImperativeHandle,
+} from 'react';
+import { useThemeClasses } from '../shared/use-theme-classes.js';
+import { cls } from '../../shared/cls.js';
+// import { useTheme } from '../shared/use-theme.js';
+import { useDarkClasses } from '../shared/use-dark-classes.js';
+import { MessageClasses } from '../../shared/classes/MessageClasses.js';
+import { MessageColors } from '../../shared/colors/MessageColors.js';
+
+const Message = forwardRef((props, ref) => {
+  const {
+    component = 'div',
+    className,
+    colors: colorsProp,
+
+    id,
+    text,
+    name,
+    type = 'sent',
+    header,
+    footer,
+    textHeader,
+    textFooter,
```

Comment:
There is no any styles for header, footer, textHeader, textFooter Message's parts. header and footer should be styles similar to `name`, textHeader and `textFooter` same as bubble text but `text-sm opacity-80`